### PR TITLE
About us screen UI skeleton

### PR DESCRIPTION
**Pull request by terencejumba**
Hey team,
I've created about us UI Skeleton
aboutt

### CODE REVIEW

File: presentation/src/main/java/com/android254/presentation/about/view/AboutScreen.kt [hunk 1]
```diff
@@ -15,20 +15,303 @@
...
 
 @Composable
 fun AboutScreen() {
-    Text(text = "About Screen")
+
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(if (isdark.value) Color.Black.copy(.6f) else Color.White.copy(.6f))
```
Comment (jumaallan) [inline presentation/src/main/java/com/android254/presentation/about/view/AboutScreen.kt:54]: @wangerekaharun  Quick one

Does the theme setup on the repo automatically choose color depending on if the device is set to dark mode or not?

Or do we need to do this?

`if (isdark.value) Color.Black.copy(.6f) else Color.White.copy(.6f)`

**Review by jumaallan — COMMENTED** [review]
@wangerekaharun Left a question for you

File: presentation/src/main/java/com/android254/presentation/about/view/AboutScreen.kt (hunk 1 above)
Comment (wangerekaharun) [reply presentation/src/main/java/com/android254/presentation/about/view/AboutScreen.kt:54]: It automatically chooses so no need to have this

Comment (wangerekaharun) [reply presentation/src/main/java/com/android254/presentation/about/view/AboutScreen.kt:54]: It automatically chooses so no need to have this

Comment (dis-jointed) [reply presentation/src/main/java/com/android254/presentation/about/view/AboutScreen.kt]: Hi team,
I've made changes to color it's now autopicking

**Comment by dis-jointed** [comment]
tr
Hi, I've made changes to color its now autopicking

**Comment by wangerekaharun** [comment]
@dis-jointed Looking at the screenshots, the screen you made does not match the visuals, there are some things missing. Can you have another look at it and make it align

File: presentation/src/main/java/com/android254/presentation/about/view/AboutScreen.kt [hunk 2]
```diff
@@ -15,20 +15,294 @@
...
+        )
+
+        aboutBody()
+
+    }
+}
+
+
+@Composable
+fun aboutBody() {
```
Comment (chepsi) [inline presentation/src/main/java/com/android254/presentation/about/view/AboutScreen.kt:71]: `aboutBody` -> `AboutBody`? Generally we should have all composables named with a titlecase.

File: presentation/src/main/java/com/android254/presentation/about/view/AboutScreen.kt [hunk 3]
```diff
@@ -15,20 +15,294 @@
...
+        aboutComponent()
+        organizingTeamComponent()
+        organisedByComponent()
+
+
+    }
+}
+
+@Composable
+fun organisedByComponent() {
```
Comment (chepsi) [inline presentation/src/main/java/com/android254/presentation/about/view/AboutScreen.kt:89]: move separate components to `com.android254.presentation.about.components` package. To ensure small files.

File: presentation/src/main/java/com/android254/presentation/about/view/AboutScreen.kt [hunk 4]
```diff
@@ -15,20 +15,294 @@
...
+fun aboutBody() {
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(start = 13.dp, end = 5.dp),
+        verticalArrangement = Arrangement.spacedBy(23.dp)
+    ) {
+
+
+        aboutComponent()
```
Comment (chepsi) [inline presentation/src/main/java/com/android254/presentation/about/view/AboutScreen.kt:80]: format file to remove unnecessary  new lines.

File: presentation/src/main/java/com/android254/presentation/about/view/AboutScreen.kt [hunk 5]
```diff
@@ -15,20 +15,294 @@
...
+        text = "About",
+        style = TextStyle(
+            color =  MaterialTheme.colorScheme.surfaceTint,
+            fontSize = 21.sp,
+            fontWeight = FontWeight.Bold
+        )
+    )
+
+    Text(
+        text = "Droidcon is a global conference focused on the engineering of Android applications. Droidcon provides a forum for developers to network with other developers, share techniques, announce apps and products, and to learn and teach.\n\nThis two-day developer focused gathering will be held in Nairobi Kenya on August 6th to 8th 2020 and will be the largest of its kind in Africa.\n\nIt will have workshops and codelabs focused on the building of Android applications and will give participants an excellent chance to learn about the local Android development ecosystem, opportunities and services as well as meet the engineers and companies who work on them.",
```
Comment (kibettheophilus) [inline presentation/src/main/java/com/android254/presentation/about/view/AboutScreen.kt:151]: Move strings to resources

File: presentation/src/main/java/com/android254/presentation/about/view/AboutScreen.kt (hunk 2 above)
Comment (terencejumba) [reply presentation/src/main/java/com/android254/presentation/about/view/AboutScreen.kt:72]: I updated this

File: presentation/src/main/java/com/android254/presentation/about/view/AboutScreen.kt (hunk 3 above)
Comment (terencejumba) [reply presentation/src/main/java/com/android254/presentation/about/view/AboutScreen.kt]: I updated this

File: presentation/src/main/java/com/android254/presentation/about/view/AboutScreen.kt (hunk 5 above)
Comment (terencejumba) [reply presentation/src/main/java/com/android254/presentation/about/view/AboutScreen.kt]: I updated this

**Comment by wangerekaharun** [comment]
> @dis-jointed Looking at the screenshots, the screen you made does not match the visuals, there are some things missing. Can you have another look at it and make it align
> 
> Also, remove the toolbar part. It's a separate UI component

@terencejumba any updates on this?

**Comment by terencejumba** [comment]
> I'm unable to put the actual images but what's in the UI can be replaced with actual images

> I moved the composables to about us 
 composable folder

> Changed about us component names to 
    start with title case

**Comment by misshannah** [comment]
@terencejumba please use this [link](https://xd.adobe.com/view/dd5d0245-b92b-4678-9d4a-48b3a6f48191-880e/screen/c87e6ded-5f2e-4f62-81f1-3940e677d79f/specs/) to put in the actual images for the UI as expected.
Use this other one for Dark mode - https://xd.adobe.com/view/5ec235b6-c3c6-49a9-b783-1f1303deb1a8-0b91/screen/fd2fa694-5139-401f-8dc0-c0cb58469aaf/specs/

**Comment by tamzi** [comment]
We will need to come back to this as we have a design system and we will navigate back to this.

**Comment by jumaallan** [comment]
@terencejumba Please rebase this PR, and we can approve it

Thank you

**Comment by jumaallan** [comment]
> @terencejumba Please rebase this PR, and we can approve it
> 
> Thank you

@terencejumba Ping :)

**Comment by terencejumba** [comment]
> > @terencejumba Please rebase this PR, and we can approve it
> > Thank you
> 
> @terencejumba Ping :)

Okay  am reading it asap

**Comment by terencejumba** [comment]
> @terencejumba Please rebase this PR, and we can approve it
> 
> Thank you

Okay am rebasing it asap

**Comment by wangerekaharun** [comment]
> > @terencejumba Please rebase this PR, and we can approve it
> > Thank you
> 
> Okay am rebasing it asap

Another ping @terencejumba

File: presentation/src/main/java/com/android254/presentation/about/components/OrganizingTeamComponent.kt [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+    LazyVerticalGrid(
+        columns = GridCells.Fixed(3),
+        modifier = Modifier
+            .fillMaxWidth()
+            .heightIn(min = 10.dp, max = 600.dp)
+    ) {
+
+        repeat(5) { index ->
+            items(count = index) {
+                TeamComponent(text = "text", index = index, subTitle = "Sub title")
```
Comment (wangerekaharun) [inline presentation/src/main/java/com/android254/presentation/about/components/OrganizingTeamComponent.kt:36]: Make a string resource too

**Comment by wangerekaharun** [comment]
Closing this due to inactivity

**Pull request closed without merging**